content/300-guides/200-deployment/110-deployment-guides/index.mdx

```diff
@@ -9,36 +9,17 @@ tocDepth: 3
 
 This page outlines the platforms and services Prisma can be deployed to.
 
-> **Note**: If you have a working example of a project using Prisma with a service not on this list, let us know!
+> **Note**: If you have a working example of a project using Prisma with a service not on this list, let us know [here]()!
 
 </TopBlock>
 
 ## Serverless and PaaS Platforms
 
 The following is an evolving list of serverless and PaaS (platform as a service) platforms that Prisma can be deployed to.
 
-### Serverless Platforms
+<Subsections />
 
-- [AWS Lambda](deploying-to-aws-lambda)
-- [Serverless Framework, AWS Lambda](deploying-to-aws-lambda)
-- [Netlify](deploying-to-netlify)
-- [Deno Deploy](deploying-to-deno-deploy)
-- [Vercel](deploying-to-vercel)
-- [Azure functions](deploying-to-azure-functions)
-- [Cloudflare Workers](deploying-to-cloudflare-workers)
-- Firebase functions
-- GCP functions
-
-### PaaS Platforms
-
-- CodeSandbox
-- [Heroku](deploying-to-heroku)
-- [Koyeb](deploying-to-koyeb)
-- Render
-- Railway
-- Fly
```

Comment:
Make sure to turn the now removed platforms into issues. This used to act as a TODO list for some previous docs writers.

Reply:
Confirming that #4721 has

- Serverless
- SST
- Netlify
- Deno Deploy
- Vercel
- Azure
- CF Workers
- GCP/Firebase
- Heroku
- Koyeb
- Render
- Railway
- Fly.io